src/components/related-grid/grid-utils.tsx

```diff
@@ -0,0 +1,111 @@
+const {PLAYER_SIZE} = KalturaPlayer.ui.components;
+
+import {GridEntry} from 'components/entry/grid-entry';
+import {MinimalGridEntry} from 'components/entry/minimal-grid-entry';
+import {MinimalNextEntry} from 'components/entry/minimal-next-entry';
+import {NextEntry} from 'components/entry/next-entry';
+import {EntryDimensions} from 'types/entry-dimensions';
+import {Sources} from 'types/sources';
+import * as styles from './related-grid.scss';
+
+const SIZE_CLASS = {
+  [PLAYER_SIZE.MEDIUM]: styles.medium,
+  [PLAYER_SIZE.LARGE]: styles.large,
+  [PLAYER_SIZE.EXTRA_LARGE]: styles.extraLarge
+};
+
+const NUM_ENTRIES = {
+  [PLAYER_SIZE.MEDIUM]: 4,
+  [PLAYER_SIZE.LARGE]: 6,
+  [PLAYER_SIZE.EXTRA_LARGE]: 8
+};
+
+const ENTRY_DIMENSIONS = {
+  [PLAYER_SIZE.MEDIUM]: {
+    imageHeight: 56,
+    contentHeight: 56,
+    width: 208
+  },
+  [PLAYER_SIZE.LARGE]: {
+    imageHeight: 98,
+    contentHeight: 49,
+    width: 174
+  },
+  [PLAYER_SIZE.EXTRA_LARGE]: {
+    imageHeight: 98,
+    contentHeight: 49,
+    width: 174
+  }
+};
+
+const ENTRY_DIMENSIONS_EXPANDED = {
```

Comment:
the name a bit confused because the grid is expended no the entry. the entry is smaller